components/AvalancheForecastZoneMap.tsx

```diff
@@ -53,154 +48,82 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
     latitudeDelta: 1.05 * region.latitudeDelta,
     longitudeDelta: 1.05 * region.longitudeDelta,
   };
+  const {isLoading, isError, data: mapLayer, error} = useMapLayer(center);
 
-  if (isList) {
-    return (
-      <>
-        {centers.map(center_id => (
-          <AvalancheForecastZoneCards key={center_id} center_id={center_id} date={date} cardStyle={styles.verticalCard} horizontal={false} />
-        ))}
-        <TouchableOpacity onPress={toggleList}>
-          <View style={styles.toggle}>
-            <FontAwesome5 name={'map'} size={32} />
-          </View>
-        </TouchableOpacity>
-      </>
-    );
-  } else {
-    return (
-      <>
-        <MapView
-          style={styles.map}
-          initialRegion={defaultRegion}
-          region={largerRegion}
-          onLayout={setReady}
-          zoomEnabled={centers.length > 1}
-          scrollEnabled={centers.length > 1}
-          provider={'google'}>
-          {isReady && centers.map(center_id => <AvalancheCenterForecastZonePolygons key={center_id} center_id={center_id} setRegion={setRegion} date={date} />)}
-        </MapView>
-        <SafeAreaView>
-          <DangerScale px="4" width="100%" position="absolute" top="12" />
-        </SafeAreaView>
-        <View style={styles.footer}>
-          {centers.map(center_id => (
-            <AvalancheForecastZoneCards key={center_id} center_id={center_id} date={date} cardStyle={styles.horizontalCard} horizontal={true} />
-          ))}
-          <TouchableOpacity onPress={toggleList}>
-            <View style={styles.toggle}>
-              <FontAwesome5 name={'list-ul'} size={32} />
-            </View>
-          </TouchableOpacity>
-        </View>
-      </>
-    );
-  }
+  return (
+    <>
+      <MapView
+        style={StyleSheet.absoluteFillObject}
+        initialRegion={defaultRegion}
+        region={largerRegion}
+        onLayout={setReady}
+        zoomEnabled={true}
+        scrollEnabled={true}
+        provider={'google'}>
+        {isReady && <AvalancheCenterForecastZonePolygons key={center} center_id={center} setRegion={setRegion} date={date} />}
+      </MapView>
+      <SafeAreaView>
+        <DangerScale px="4" width="100%" position="absolute" top="12" />
+      </SafeAreaView>
+
+      {isLoading && <FeatureTitleBlack>Loading...</FeatureTitleBlack>}
```

Comment:
Why do none of these conditionally-rendered components require a `<SafeAreaView>`?

Reply:
because they're absolutely positioned, in the center or at the bottom of the screen

Reply:
Gotcha - if someone turned their phone upside down, wouldn't the "bottom" have the iOS notch?

Reply:
Oh - gotcha, had no idea that was an opt-in thing!